react/tic-tac-toe/src/app/screens/Login/components/LoginInput/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+function LoginInput({ input, type, meta: { error, submitFailed } }) {
```

Comment:
Why not move this to common component like 

```
| - app
     | - components
              | - Input
```

That way it works not only for Login inputs but whatever Fom that needs an Input. By the way, remember to add the label for this.